### ISSUE TITLE
Readme example is not working

### ISSUE THREAD

**Issue by Spacehug** [perdy/starlette-api#16]
Copy-pasting the example from readme produces this:

```
INFO: ('127.0.0.1', 33118) - "GET / HTTP/1.1" 500
ERROR: Exception in ASGI application
Traceback (most recent call last):
  File "<venv>/lib/python3.7/site-packages/uvicorn/protocols/http/httptools_impl.py", line 372, in run_asgi
    result = await asgi(self.receive, self.send)
  File "<venv>/lib/python3.7/site-packages/starlette/middleware/errors.py", line 125, in asgi
    raise exc from None
  File "<venv>/lib/python3.7/site-packages/starlette/middleware/errors.py", line 103, in asgi
    await asgi(receive, _send)
  File "<venv>/lib/python3.7/site-packages/starlette/exceptions.py", line 74, in app
    raise exc from None
  File "<venv>/lib/python3.7/site-packages/starlette/exceptions.py", line 63, in app
    await instance(receive, sender)
  File "<venv>/lib/python3.7/site-packages/starlette_api/routing.py", line 168, in awaitable
    injected_func = await app.injector.inject(endpoint, state)
  File "<venv>/lib/python3.7/site-packages/starlette_api/injection.py", line 173, in inject
    state[output_name] = resolver(**kw)
  File "<venv>/lib/python3.7/site-packages/starlette_api/components/validation.py", line 79, in resolve
    query_params = validator().load(dict(query_params), unknown=marshmallow.EXCLUDE)
AttributeError: module 'marshmallow' has no attribute 'EXCLUDE'
```

_UPD_: I used `starlette-api[full]` for install

**Comment by perdy** [comment]
It seems that you're using an older version of marshmallow (2.x) and it's required a 3.x version.

**Comment by Spacehug** [comment]
I have no `marshmallow` defined in Pipfile, which means I got it installed with `starlette-api`, and that means that the project dependencies are off somehow. The version of marshmallow specified in `pyproject.toml` is ">2" which means 2.19.1 suits fine.

**Comment by perdy** [comment]
It should be fixed with version 0.6.4
